### ISSUE TITLE
space-after-keywords never should ignore else

### ISSUE THREAD

**Issue by 0x80** [eslint/eslint#3003]
I would like to enforce this style:

```js
if(foo) {
}
else if(foo) {
}
else {
}
```

Using space before { but not between keywords and (. This code with "never" gives the error:

```Keyword else must not be followed by whitespace```

**Comment by ilyavolodin** [comment]
Could you post the whole error message including line/column numbers? Also, which version of ESLint are you using?

**Comment by gyandeeps** [comment]
Closing because of no response.
Feel free to reopen with more info.

**Comment by yordis** [comment]
@gyandeeps @ilyavolodin   this will fail, the else statement should't be include because we don't have parenthesis after, we have brackets. 
 
The column doesn't not match because I didn't take the message from that example, I hope you understand.

**Packages**
- "eslint": "^1.2.1"
- "babel-eslint": "^4.1.0",

```
 25:12  error    Keyword "else" must not be followed by whitespace  space-after-keywords
```

```js
if(...) {
  // ...
} else {
  // ...
}
```

**Comment by gyandeeps** [comment]
@yordis I am not sure what line 25 refers too.
I tried the your example with default parse, it works fine.
You can try it here: http://eslint.org/demo/

**Comment by yordis** [comment]
@gyandeeps I said this about the lines
> The column doesn't not match because I didn't take the message from that example, I hope you understand.

because I can't use non-open source code here.

@gyandeeps notice I am using ES6 features too that's why I include babel-eslint there

**Comment by BYK** [comment]
@yordis - This is a different issue so please file a new ticket. Also it is quite hard to understand what you actually want. I tried the following code in the demo:

```js
/* eslint space-after-keywords: [2, "never"] */
if(a) {
  // ...
} else {  // 4:8 - Keyword "else" must not be followed by whitespace. (space-after-keywords)
  // ...
}
```

And got the error on the marked line, which is expected. That said if you want that changed, please file a new ticket, state your problem clearly and mention any solutions you have in mind so others can weigh in and discuss.

**Comment by yordis** [comment]
@BYK sorry dude my English is not good enough sometimes. 

Why is expected because basically that mean I have to do ```else{``` and that's weird for the else statement

**Comment by BYK** [comment]
Because the rule is called `space-after-keywords` and `else` is a keyword and you set it to `never` so the rule does not expect a space after `else.` I think [space-before-blocks](http://eslint.org/docs/rules/space-before-blocks) is the rule you are looking for.

If you disagree, please file a new ticket.
